### DISCUSSION TITLE
Gamepad Support

### DISCUSSION THREAD

**Discussion by CyberManMatt** [Schneegans/Fly-Pie#224] [Ideas & Feature Discussions]
This may be a stretch, but a suggestion would be to allow the user to use a gamepad to control fly-pie.

I have a home theater pc with Fly-Pue installed. It works really great, but I think it would be awesome to have a gamepad as an input.

**Comment by Schneegans** [comment]
While I think that this is a good idea, I think that I will not have the time to work on this in the foreseeable future. However, maybe someone else wants to implement this - hence I'll transfer this to an issue so that maybe more potential contributors are aware of this idea!

Please continue the discussion in #226.
